collectionDBConnection.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+$db = new PDO ('mysql:host=db; dbname=OfficeQuotesDB', 'root', 'password');
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
Would suggest wrapping this in a function that just returns the PDO connection. You can then call the function from your main collection.php file to get a connection

That way you can re-use this when it comes to adding to the collection without automatically running a query to load the quotes.